setup.py

```diff
@@ -1,65 +1,72 @@
-from glob import glob
+from __future__ import absolute_import, division, print_function, unicode_literals
+
 import os
 import sys
+from glob import glob
 
-from setuptools import find_packages, setup, Extension
+from setuptools import Extension, find_packages, setup
 
-long_description = 'Scout Application Performance Monitoring Agent - https://scoutapp.com'
-if os.path.exists('README.md'):
-    long_description = open('README.md').read()
+long_description = (
+    "Scout Application Performance Monitoring Agent - https://scoutapp.com"
+)
+if os.path.exists("README.md"):
+    long_description = open("README.md").read()
 
 # Try to compile the extensions, except for platforms or versions
 # where our extensions are not supported
 compile_extensions = True
 
 setup_args = {
-      'name': 'scout_apm',
```

Comment:
I explicitly used single quotes unless a string needs interpolation. Ultimately it probably doesn't matter, but are we switching to double quotes just because of historical preference?

Reply:
There are two common conventions in Python-land:

A. Single quotes for "technical strings" (~ Ruby symbols) and double quotes for "human-readable strings" (~ could be translated)
B. Double quotes for everything (except perhaps strings containing double quotes, in order to avoid backslash-escaping)

By default `black` uses option B. I used to be a fan of option A; `black` made me realize it creates mental overhead for almost no practical benefits. I'm now a convert to option B aka "just don't think about it".

I can re-run `black` with the `-s` option which preserves the choice of single or double quotes if you'd like (and rebase the rest of the pull request, which is non-trivial work because there will be lots of conflicts).

Conclusion: just tell me if this convinces you to go with double quotes or if you'd like me to preserve the original quote style.

Reply:
I don't care one bit, beyond not creating too much PR noise due to disagreement about it. Do the change here, and it looks like unit tests will enforce this style going forward?

Reply:
Yes, `make style` will auto-format the code and `tox -e check` will enforce the auto-formatting going forwards.